docs/www/configuration.md

```diff
@@ -462,6 +462,13 @@ pandaproxy_client:
   scram_password: ""
 
 rpk:
+  # Add flags to run rpk with specific parameters
```

Comment:
we should add a link to the configuration.cc and say any additional command line flag can be passed there.